src/content/guides/farmlands-portal-introduction.mdx

```diff
@@ -0,0 +1,200 @@
+
+In some cases, a user may be requested to select a business location as the default.  This ensures all authorises are loaded against that location and can assist when searching for authorisation history.
+
+> We recommend bookmarking the [Centrapay Business Portal](https://app.centrapay.com/business) for easy access.
+
+### Dashboard
+
+The Dashboard is an overview of Centrapay Business Account and will be your main screen for generating an authorisation or releasing a previous Farmlands Authorisation.
+
+<img src="/business-portal-dashboard.png" alt="The dashboard of the centrapay business portal" style=""/>
+
+## Farmlands Authorisations
+
+The Centrapay Business Portal provides a real-time authorisation of a Farmlands Card when presented as payment for the purchase of goods or services. It validates that the Card is active and there is a sufficient available balance to process the transaction.
+
+### Authorisation Responses
+
+|          Type          |                                                                                                        Description                                                                                                         |
+| ---------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Authorisation Approved | Funds have now been placed on hold onto the customer account, pending the receipt of the Card Partner’s invoice within the agreed period.                                                                                  |
+| Pending                | Card Partner creates the request, but the authorisation is not yet complete. This authorisation is not approved.                                                                                                           |
+| Check ID               | This response may occur for a high-value sale. This means the transaction is pending authorisation until the Card Partner confirms they have authenticated the customer’s photo ID against the name on the Farmlands Card. |
+| Declined               | The authorisation is declined due to an invalid card number, a credit limit exceeded, or Card Partner is unable to verify a high-value transaction. Refer to Farmlands Contact Centre 0800 200 600                         |
+| Release                | The funds “on hold” are cancelled by Card Partner. This may occur when a Card Partner is unable to fulfil an order. (This is not used for a refund).                                                                       |
+
+### Create a Farmlands Authorisation
+
+1. Select Create Authorisation Request from the main dashboard.
+
+    <img src="/farmlands-payment-create-authorization-request.png" alt="Create a Farmlands authorisation request on the Centrapay business portal" style=""/>
```

Comment:
You should be able to use Markdown syntax for these images rather than embedding HTML here

`![Create a Farmlands authorisation request on the Centrapay business portal](/farmlands-payment-create-authorization-request.png)`

We aimed to keep our content as close to raw markdown as possible. In the future when there's a CMS on top of our site, we want others to be able to update images (although admittedly I'm not entirely sure how that would work as we haven't selected a CMS yet)

Reply:
I can keep it close to markdown. However, I think you'll find where ever markdown is accepted so is HTML.
Markdown is easier to read and so it's preferred for content.

When you use a CMS images are hosted somewhere then in either the build step or server-side rendering you retrieve that image from the server.